mackup/applications/rtx.cfg

```diff
@@ -0,0 +1,14 @@
+[application]
+name = rtx
+
+[configuration_files]
+.config/rtx
```

Comment:
This should be moved in a `xdg_configuration_files` section like explained at https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-using-the-xdg-directory

Reply:
I will make this change to `asdf` as well.